helpers/helpers.go

```diff
@@ -32,3 +38,23 @@ func LoadConfig(file string) (models.Config, error) {
 	}
 	return cfg, err
 }
+
+func NewResponseBuilder(status bool, message string, obj interface{}) (jsoned string, err error) {
```

Comment:
"jsoned" => if not used, I think its better to delete this

Reply:
i forget about that and ill change it soon